### PR TITLE
Document versions policy stuff

### LINKED ISSUE #70: Golang 1.1 not supported anymore?
Hello. I'm using Travis CI for testing on macos. They uses gimme to fetch different Golang versions.

My latest build runs into error and here is the message:

```
Installing gimme from https://raw.githubusercontent.com/travis-ci/gimme/v1.0.0/gimme

115.84s$ GIMME_OUTPUT=$(gimme 1.1) && eval "$GIMME_OUTPUT"

I don't have any idea what to do with '1.1'.

  (using type 'auto')

The command "GIMME_OUTPUT=$(gimme 1.1) && eval "$GIMME_OUTPUT"" failed and exited with 1 during .
```

So apparently `gimme 1.1` some how fails and the build fails.

I couldn't find the "supported go version" in gimme's site. Does gimme currently support Go 1.1?

### PR DESCRIPTION
**Pull request by philpennock**
Closes #70.

### CODE REVIEW

**Comment by philpennock** [comment]
NB: Do not merge until #130 is approved and merged into this; GitHub handles stacked PRs very badly.
(In turn, #130 needs #131 merged into it)

**Review by AlekSi — APPROVED** [review]

**Comment by AlekSi** [comment]
Edited first comment to convert a text reference to a link.

**Review by meatballhat — APPROVED** [review]
:trumpet: :turtle:

**Pull request merged**
